### DISCUSSION TITLE
Run command in new pane

### DISCUSSION THREAD

**Discussion by rcortezk9** [wezterm/wezterm#4466] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Mutter

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Newbie question here. Is there a setup where a new pane opens up where it runs the cargo run command?

**Comment by bew** [comment, marked as answer]
Did you try https://wezfurlong.org/wezterm/config/lua/keyassignment/SplitPane.html ?
